kafka-client/src/main/java/dev/responsive/kafka/internal/db/mongo/MongoWindowedTable.java

```diff
@@ -457,13 +459,37 @@ public KeyValueIterator<WindowedKey, byte[]> backFetch(
   @Override
   public KeyValueIterator<WindowedKey, byte[]> fetchRange(
       final int kafkaPartition,
-      final Bytes fromKey,
-      final Bytes toKey,
+      final Bytes keyFrom,
```

Comment:
question about the state store api in general - what are these cross-key APIs actually used for? All the kafka streams processors are scoped to one key afaik.